src/api/nano.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { createRequestInstance } from './axiosInstance';
+import { NanoRequest404Error, NanoRequestError } from '../errors';
+
+/**
+ * Api calls for nano contracts
+ *
+ * @namespace ApiNanoContracts
+ */
+
+const ncApi = {
+  /**
+   * Call get nano contracts state API
+   *
+   * @param id Nano Contract ID
+   * @param fields Array of fields to get state
+   *
+   * @return {Promise}
```

Comment:
I removed the return types and added some missing parameters in the jsdocs.

In the next PR I created all response types (https://github.com/HathorNetwork/hathor-wallet-lib/pull/604/commits/d9771e6ed3d793097f4fbc01e59cae8d789383d6) because there I already have a file for nano types.

After this PR is merged I will add those types to the nano contract API methods. There's already a TODO in the other PR description.

What do you think? (this is for all the comments)